src/about-menu/open-about-menu.ts

```diff
@@ -49,14 +49,14 @@ export function openAboutWindow() {
 			const versionText = sprintf( __( 'Version %s' ), packageJson );
 			const studioByWpcomText = escapeSingleQuotes( __( 'Studio by WordPress.com' ) );
 			const shareFeedbackText = escapeSingleQuotes( __( 'Share Feedback' ) );
-			const demoSitesText = escapeSingleQuotes( __( 'Demo sites powered by' ) );
+			const previewSitesText = escapeSingleQuotes( __( 'Preview sites powered by' ) );
```

Comment:
Could we keep variables and filenames as before? Let's change just the strings that are user facing.

```suggestion
			const demoSitesText = escapeSingleQuotes( __( 'Preview sites powered by' ) );
```

Reply:
I have put them all back in https://github.com/Automattic/studio/pull/933/commits/8135840b6fb0a9fccea8da57f3daa39b2a3deeb9,